Dockerfile-dev

```diff
@@ -8,30 +7,3 @@ RUN apt-get update \
     && pip install pynacl \
     && apt-get autoremove \
     && apt-get clean
-
```

Comment:
Why are we removing everything from here?

Reply:
We shouldn't be. Vansh and I were screwing around trying to get my test env working, and I forgot to put these lines back in...